packages/api/src/routes/auth/discordAuthMe.ts

```diff
@@ -57,7 +57,8 @@ export default class extends Route<GetDiscordAuthMeResult, never> {
 			discriminator,
 			avatar,
 			guilds: await Promise.all(
-				guilds.map(async ({ id, name, icon }) => {
+				// @ts-expect-error TS2339
```

Comment:
```suggestion
				// @ts-expect-error - Missing discord-api-types
				// TODO: Remove once upstream has the types
```

`TODO:` comments show up as warnings in CI so we don't forget.